src/table/components/TableWrapper.jsx

```diff
@@ -140,9 +140,8 @@ export default function TableWrapper(props) {
             `${columns.length}`,
           ])}
         >
-          <TableHeadWrapper {...props} setFocusedCellCoord={setFocusedCellCoord} focusedCellCoord={focusedCellCoord} />
+          <TableHeadWrapper setFocusedCellCoord={setFocusedCellCoord} focusedCellCoord={focusedCellCoord} />
```

Comment:
same here as well. 
if we move the logic for focused cell coords into the hook inside of context, we wont pass props to the functions like this one